test/cppsuite/test_harness/util/api_const.h

```diff
@@ -74,7 +74,12 @@ static const char *STABLE_TS = "stable_timestamp";
 static const char *STATISTICS_LOG = "statistics_log=(json,wait=1)";
 
 /* Test harness consts. */
-static const char *DEFAULT_FRAMEWORK_SCHEMA = "key_format=S,value_format=S";
+#define BLKCMP_PFX "block_compressor="
+#define SNAPPY_BLK BLKCMP_PFX "snappy"
+#define EXTPATH "../../ext/compressors/"
+#define SNAPPY_EXT ",extensions=(" EXTPATH "snappy/.libs/libwiredtiger_snappy.so)"
```

Comment:
The reference to `.libs` is a LibTool thing and will break with our CMake builds. Not sure whether we need to address this now.

I can pull up an old PR that @alisonfel made where she fixed references to `.libs`. Can't remember how it was solved off the top of my head.

Reply:
Interesting, it seems to be that way in wtperf and format.

Reply:
@tetsuo-cpp Yep, `.libs` is libtool specific. To get around this specifically in format I updated the CPP definition like so:
https://github.com/wiredtiger/wiredtiger/blob/23bfdf76113c30db15f26e62f28ab554001ab82c/test/format/format.h#L42-L60

and in the CMake file, I passed the CPP definitions as compiler arguments:
https://github.com/wiredtiger/wiredtiger/blob/23bfdf76113c30db15f26e62f28ab554001ab82c/test/format/CMakeLists.txt#L58-L65

This was to be compatible across both CMake & Autconf (wanted to avoid making significant changes to autoconf or format).

Reply:
9949c13

Reply:
Thanks, @alisonfel. At the moment, we're planning to give users to the option to run with Snappy from their stress testing configuration.

I provide `SNAPPY_PATH` from CMake if we have Snappy enabled and then construct the path with `.libs` only if it hasn't already been defined.